src/components/Lend/LoanSearch/LoanSearchInterface.vue

```diff
@@ -15,15 +18,31 @@
 						{{ null }} <!-- Hide title text -->
 					</template>
 					<loan-search-filter
+						style="min-width: 285px;"
```

Comment:
The two `min-width` entries here could be `tw-min-w-[285px]` or are we preferring the inline style in this case?

Reply:
We prefer inline styles for items that don't have an automated tailwind unit.